source/overloading.tex

```diff
@@ -1479,8 +1479,11 @@
 \pnum
 When objects of non-aggregate class type \tcode{T} are
 list-initialized such that \ref{dcl.init.list} specifies that overload resolution
-is performed according to the rules in this subclause, overload resolution selects the
-constructor in two phases:
+is performed according to the rules in this subclause
+or when forming a list-initialization sequence according to \ref{over.ics.list},
+overload resolution selects the constructor in two phases,
+except that the first phase is omitted
+if the initializer list has no elements and \tcode{T} has a default constructor.
```

Comment:
I think it would be clearer to inline this into the first bullet:

> * If the initializer list is not empty or `T` has no default constructor, overload resolution is first performed where the candidate functions are the initializer-list constructors ([dcl.init.list]) of the class `T` and the argument list consists of the initializer list as a single argument.
> * Otherwise, or if no viable initializer-list constructor is found, overload resolution is performed again, where the candidate functions are all the constructors of the class `T` and the argument list consists of the elements of the initializer list.

Reply:
Yes, that reads better.